src/init/wazuh/wazuh.sh

```diff
@@ -197,4 +197,12 @@ WazuhUpgrade()
         rm -f $DIRECTORY/etc/ossec-init.conf
         rm -f /etc/ossec-init.conf
     fi
+
+    # Replace and delete ossec group together with ossec users
+    OSSEC_GROUP=ossec
+    if (grep "^ossec:" /etc/group > /dev/null 2>&1) || (dscl . -read /Groups/ossec > /dev/null 2>&1)  ; then
+        find $PREINSTALLEDDIR -group $OSSEC_GROUP -user root -exec chown root:wazuh {} \;
```

Comment:
Let's replace `$DIRECTORY` by `$PREINSTALLEDDIR` in this whole function, please. That way it will work in versions higher than 4.2 whenre the `ossec_init.conf` file is deprecated.